source/standards/principle-least-access.html.md.erb

```diff
@@ -43,6 +43,6 @@ If you’re using the gds-users account to log into your AWS accounts, you shoul
 
 - [NCSC - 10 steps to cyber security](https://www.ncsc.gov.uk/collection/10-steps)
 - [NCSC - privileged user management](https://www.ncsc.gov.uk/guidance/introduction-identity-and-access-management#section_6)
-- [NIST Special Publication 800-53 - AC-6 least privilege][polp]
+- [NCSC - preventing lateral movement][polp]
 
-[polp]: https://csrc.nist.gov/Projects/risk-management/sp800-53-controls/release-search#!/control?version=5.1&number=AC-6
```

Comment:
I don't have the context here but this link appears to resolve fine. If we're changing the link just because we think the NCSC guidance is better or more appropriate, should we split this change out as it's not broken?

Reply:
Yep, I changed this in an early draft to get it working and didn't change it back, reverted in next change.